app.js

```diff
@@ -0,0 +1,276 @@
+'use strict'
+
+let seattleSales = document.getElementById('seattle');
+// console.log(cookieElem);
+let timeHours = ['6am','7am','8am','9am','10am','11am','12pm','1pm','2pm','3pm','4pm','5pm','6pm','7pm'];
+
+function custHr(min, max) {
+    return Math.floor(Math.random() * (max - min + 1) + min);

```

Comment:
I REALLY like this, functions are great for keeping you from having to write the same line of code repeatedly, and since this is out of any objects, you don't have to repeat the function.